lib/stm32/fdcan.c

```diff
@@ -0,0 +1,891 @@
+/** @defgroup fdcan_file FDCAN peripheral API
+ * @ingroup peripheral_apis
+ * 
+ * @brief <b>libopencm3 STM32G4xx CAN</b>
```

Comment:
fix this "G4" bit, it's not.  Also, this sort of common file normally goes into the common directory, not the root.  (the bxcan file is iirc, the _only_ file that isn't in that style, and not a good example to follow :)